spec/features/logging/view_auth_requests_for_a_location_spec.rb

```diff
@@ -11,6 +11,19 @@
       click_on 'Go to search'
     end
 
+    context 'With a new set of locations' do
```

Comment:
Not a very descriptive context. Doesn't fit in with the rest of the spec. I wonder if we need this context at all.